core/src/main/scala/kafka/log/LogCleanerManager.scala

```diff
@@ -114,6 +116,27 @@ private[log] class LogCleanerManager(val logDirs: Array[File], val logs: Pool[To
   }
 
   /**
+    * Find any logs that have compact and delete enabled and that have segments
+    * ready for deletion
```

Comment:
The comment is not accurate now since it doesn't check for segments ready for deletion.